x-pack/plugins/synthetics/public/apps/synthetics/components/common/monitor_test_result/result_details.tsx

```diff
@@ -130,7 +130,6 @@ export const StepMetrics = ({ step }: { step: JourneyStep }) => {
   return (
     <EuiDescriptionList
       compressed={true}
-      gutterSize="s"
```

Comment:
shouldn't this be renamed instead of removing?

Reply:
Hi @shahzad31. This breaking change had two parts, renaming the prop to `rowGutterSize` and remapping the default gutter size to `s`. That's why I removed the `gutterSize="s"` prop entirely instead of removing it.